modules/ossm-enabling-jaeger.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-enabling-tracing_{context}"]
+= Enabling Tracing
+
+You enable tracing by specifying a tracing type and a sampling rate.
+
+.Default `all-in-one` Jaeger parameters
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 100
+    type: jaeger
+----
+
+Currently the only tracing type that is supported is `jaeger`.   Jaeger is enabled by default.  To disable tracing, set `type` to `None`.
```

Comment:
Type is also case sensitive, must be 'Jaeger' instead of 'jaeger'

Reply:
Fixed.